### PR TITLE
Add Discord notification

### PR DESCRIPTION
**Pull request by anthonygraignic**
Hey, I'm a great fan of you project, thank you so much !
I needed an external status page and hosting it with cf-workers is an amazing idea !

I also prefer Discord over Slack for team communication so I added the method & secret in this PR.

It renders like this:
image

Don't hesitate to tell me if something is wrong ;)
(and please squash my commits as they were for test purposes)

### CODE REVIEW

File: wrangler.toml [hunk 1]
```diff
@@ -5,7 +5,7 @@ type = "webpack"
 webpack_config = "node_modules/flareact/webpack"
 
 [triggers]
-crons = ["* * * * *"]
+crons = ["*/2 * * * *"] # Workers KV free tier
```
Comment (eidam) [inline wrangler.toml:8]: What about adding the KV free tier as a commented option?

**Comment by eidam** [comment]
Hey, thank you for your kind words and great PR! I was planning to add Discord myself so its great to see someone else did that. :) It looks good, just added s small comment

File: wrangler.toml (hunk 1 above)
Comment (anthonygraignic) [reply wrangler.toml]: Oupsie, solved !

**Comment by eidam** [comment]
Thank you! :)

**Pull request merged**